core/src/main/scala/cats/Reducible.scala

```diff
@@ -187,3 +187,93 @@ import simulacrum.typeclass
   override def maximumOption[A](fa: F[A])(implicit A: Order[A]): Option[A] =
     Some(maximum(fa))
 }
+
+/**
+ * This class defines a `Reducible[F]` in terms of a `Foldable[G]`
+ * together with a `split` method, `F[A]` => `(A, G[A])`.
+ *
+ * This class can be used on any type where the first value (`A`) and
+ * the "rest" of the values (`G[A]`) can be easily found.
+ */
+abstract class NonEmptyReducible[F[_], G[_]](implicit G: Foldable[G]) extends Reducible[F] {
```

Comment:
Unfortunately adding `val` didn't seem to fix the issue, I'll have to look into it further 🤔